docs/pages/workflow/already-used-react-native.md

```diff
@@ -26,9 +26,9 @@ Expo offers a shared configuration file we call a manifest. Typically you'd upda
 
 - Configuration that you would typically do inside of your Xcode / plist files or Android studio / xml files is handled through `app.json`. For example, if you want to lock the orientation, change your icon, customize your splash screen, add/remove permissions and entitlements (in standalone apps), configure keys for Google Maps and other services, you set this in `app.json`. [See the guide here](../workflow/configuration.md).
 
-In the managed workflow, you can share your app with anyone, anywhere in the world while you're working through the Expo client [(available on the App / Play Store)](https://expo.io). Scan a QR code, or enter in a phone number and we'll send you a link that will instantly load your app on your device.
+In the managed workflow, you can share your app with anyone, anywhere in the world while you're working through the Expo Go app [(available on the App / Play Store)](https://expo.io). Scan a QR code, or enter in a phone number and we'll send you a link that will instantly load your app on your device.
 
-- Instead of having to sign up several external testers through iTunes connect, you can easily have them download the Expo client app and immediately have a working version on their phone.
+- Instead of having to sign up several external testers through iTunes connect, you can easily have them download the Expo Go app and immediately have a working version on their phone.
```

Comment:
```suggestion
- Instead of having to sign up several external testers through App Store Connect, you can easily have them download the Expo Go app and immediately have a working version on their phone.
```